src/app/tests/suites/certification/Test_TC_S_2_3.yaml

```diff
@@ -389,18 +381,14 @@ tests:
                                         AttributeID: 0x0000,
                                         AttributeValue: 0x64,
                                     },
-                                    {
-                                        AttributeID: 0x0001,
-                                        AttributeValue: 0x00,
-                                    },
                                 ],
                         },
                     ]
 
     - label:
           "Step 3b: TH sends a ViewScene command to DUT with the GroupID field
           set to G1 and the SceneID field set to 0x01."
-      PICS: S.S.C41.Rsp && PICS_SDK_CI_ONLY
+      PICS: (!S.S.C41.Rsp) && PICS_SDK_CI_ONLY
```

Comment:
Neither the old nor the new PICS seems to match the test plan?  But the test plan also seems pretty weird, in that it does EnhancedViewScene and then does ViewScene for the same exact scene....

Reply:
As per above, I can add a check for S.S.C01.R, but executing the two tests seems redundant as they write the same scene to memory and will overwrite each other.

Reply:
What this needs is a test plan issue, not unilateral changes to the official certification tests to make them not match the test plan...

Reply:
Issue [here](https://github.com/CHIP-Specifications/chip-test-plans/issues/3509), comment added